db/database.go

```diff
@@ -48,6 +48,14 @@ const (
 	DBCompactRunning
 )
 
+const (
+	ImportEvent uint32 = iota
+	SGWriteEvent
+	BlipWriteEvent
+)
+
+var eventType uint32
+
```

Comment:
I think you want to use the standard pattern for an enum type.  In addition, since we've already got the concepts of events in SGW, the name should be a bit more specific.
```suggestion
const (
	ImportEvent docUpdateType = iota
	SGWriteEvent
	BlipWriteEvent
)

type docUpdateType uint32

```

Reply:
(Made a comment elsewhere about renaming the three types as well)